nncf/experimental/tensorflow/nncf_context.py

```diff
@@ -0,0 +1,135 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import Dict
+from typing import Any
+import threading
+
+
+_NNCF_CONTEXT = threading.local()
+
+
+def get_nncf_context():
+    """
+    Returns current active `NNCFContext`.
+
+    :return: NNCF context.
+    """
+    nncf_context = getattr(_NNCF_CONTEXT, 'nncf_context', None)
+    if nncf_context is None:
+        nncf_context = NNCFContext()
+        setattr(_NNCF_CONTEXT, 'nncf_context', nncf_context)
+    return nncf_context
+
+
+class NNCFContext:
```

Comment:
In PT, we have "TracingContext". Perhaps we should rename this to "WrappingContext" or "TracingContext" as well for consistency.

Reply:
Thanks for the comment! I renamed the `NNCFContext` to the `TracingContext` for consistency.